### ISSUE TITLE
Include compatibility with `ses-` tag

### ISSUE THREAD

**Issue by valosekj** [valosekj/dicom-to-bids-tutorial#1]
Since subjects might have multiple sessions (i.e., `ses-01`, `ses-02`, etc.), they also might have multiple source IDs, each one associated with a specific session. Thus, `source_id` in [participants.tsv](https://github.com/valosekj/dicom-to-bids-tutorial/blob/main/participants.tsv) should be replaced by something like `source_id_ses-01`, `source_id_ses-02`, etc.

**Comment by mguaypaq** [comment]
I guess this only works nicely if all subjects have the same sessions? If there are only a few exceptions, I guess the script could handle `N/A` values, but otherwise I would suggest using a separate `.tsv` file: the columns would be `subject_id`, `session_id`, `source_id`, and the primary key would be `source_id` (meaning that `subject_id` and `session_id` values can be repeated, but there would be a single row containing each distinct `source_id`).

**Comment by valosekj** [comment]
> I would suggest using a separate `.tsv` file

Do you mean to have multiple `participants.tsv` files? I am not sure if this is allowed by BIDS.

I meant something like this:

participant_id | source_id_ses-01 | source_id_ses-02 | sex | age
-- | -- | -- | -- | --
sub-001 | 54321 | 74893 | M | 30
sub-002 | 65432 | 83932 | F | 20
sub-002 | 65432 | n/a | F | 20

**Comment by mguaypaq** [comment]
The extra `.tsv` file could live under `code/`, next to the script which uses it, maybe. I'm just pointing out that, for `participants.tsv`, the column `participant_id` is a primary key, but the conversion script would work best with a file where `participant_id` is _not_ a primary key. Because the script needs to run `dcm2bids` once for each _session_, not once for each _participant_.

**Comment by valosekj** [comment]
> Because the script needs to run `dcm2bids` once for each _session_, not once for each _participant_.

Yes, this is something I am aware of. The [current script](https://github.com/valosekj/dicom-to-bids-tutorial/blob/main/wrapper_dcm2bids.sh) must be modified to be compatible with this scenario. In other words, the script would have to loop across participants and also across each participant's sessions. But I believe it is not a big problem. I use the same approach in one of my internal studies, and it works fine.

> Otherwise we'd have to periodically add columns which are almost completely N/A, and modify the script each time to take the new column into account.

You are right. For each new session, a new column would have to be added. But, usually, subjects have two or three sessions. Thus, we would modify the `participants.tsv` only once or twice. And also, usually, the study design is established before the study starts. In other words, we know in advance how many sessions we will have.
